bigbluebutton-html5/imports/api/common/server/helpers.js

```diff
@@ -10,8 +13,17 @@ export const indexOf = [].indexOf || function (item) {
   return -1;
 };
 
-// used in 1.1
-export const inReplyToHTML5Client = arg => arg.routing.userId === 'nodeJSapp';
+export const skipFlashDirectEvent = fn => (message, ...args) => {
```

Comment:
I would consider renaming the function to `processForHTML5UsersOnly`. What do others think in terms of naming?